docs/sources/alerting/set-up/provision-alerting-resources/http-api-provisioning/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/developers/http_api/alerting_provisioning/
+description: Create and manage alerting resources using the HTTP API
+keywords:
+  - grafana
+  - alerting
+  - alerting resources
+  - provisioning
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Use the HTTP API to manage alerting resources
```

Comment:
just a question on choice of using manage here - why not provision like the other two?

Reply:
First, I found the **The HTTP API and File Provisioning use distinct formats and aren’t interchangeable** --> https://github.com/grafana/grafana/issues/81387

Then, I clarified this on this PR: 

>    The JSON output from the majority of Alerting HTTP endpoints is not compatible for provisioning via configuration files. Instead, use the [Export Alerting endpoints][alerting_export_http] to return or download the alerting resources in provisioning format.

and

> Note that most Alerting endpoints return a JSON format that is not compatible for provisioning via configuration files, except the ones listed below.

Why did I use `manage` for the HTTP API?

I interpret Provisioning as setting up and managing the infrastructure components in an automated way via Code during the initialization process. With the API, users can manage individual components directly and be used for other cases (not only Provisioning)

I am unsure about this. And perhaps, this distinction makes it more complicate. Would be good to more feedback.

Reply:
ok definitely something for the wider team then